data/json/mutations/mutations.json

```
@@ -842,7 +842,7 @@
     "id": "XXXL",
     "name": { "str": "Weight: XXXL" },
     "points": -1,
-    "description": "You gained an awful lot of weight before Cataclysm.  Whatever the reason, your bloated figure is now an extra burden, when mobility is a key to survival.  You need to go on a diet or die trying.",
+    "description": "You gained an awful lot of weight before the Cataclysm.  Whatever the reason, now that mobility is key to survival, your bloated figure is now an extra burden.  You need to go on a diet or die trying.",
```

"now" used two times. Is that intentional?

---

```suggestion
    "description": "You gained an awful lot of weight before the Cataclysm.  Whatever the reason, now that mobility is key to survival your bloated figure is an additional burden.  You need to go on a diet or die trying.",
```
While the parallelism of "diet" and "die" is nice (I am remembering a Garfield comic strip...), "lose weight" may be preferable to "go on a diet".

---

I didn't think of it before, but I agree, to align with XS' usage.